bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/BigtableResultScannerAdapter.java

```diff
@@ -56,6 +56,10 @@ public void close() {
           throw Throwables.propagate(ioe);
         }
       }
+
+      public boolean renewLease() {
```

Comment:
I specifically don't want to add @Override, since I want this to be backwards compatible with hbase 1.0.1 and 1.1.1.